### DISCUSSION TITLE
Please add support for imxrt1052 board. EC500

### DISCUSSION THREAD

**Discussion by ashokmanker** [grblHAL/core#531] [Ideas]
Please add support for iMXrt1052 board. Perticularly EC500 which is freely available on net. Has plenty of i/o and spindle support alongwith regular MPG pendant support. It will be great board for CNC with GRBLHAL. Persons who are unhappy with mach3 or difficult to configure LinuxCNC can easily migrate to grblhal. For starting you can have reference to remora linuxcnc git for pin layout and configuration.

**Comment by terjeio** [comment]
Do you think they will provide a controller and pay me for adding it?

**Reply by ashokmanker** [reply]
No, they will not provide any controller and they will pay nothing to anyone. 
Can you guide me to getting started? Since you have repo of imxrt1062. I might try to port it to imxrt1052. At least I can try.

**Reply by terjeio** [reply]
The iMXRT1062 driver is based on the Arduino framework (Teensy) so you may have to port that as well?
Better/easier to switch to Atmels own?
Without doing some research I do not know what is your best option.
